src/backend/src/utils/slack.utils.ts

```diff
@@ -70,6 +70,37 @@ export const sendSlackTaskAssignedNotification = async (slackId: string, task: T
   await sendMessage(slackId, msg, link, linkButtonText);
 };
 
+/**
+ * Send a notification to users that a reimbursement request is created on Slack
+ * @param slackId the slack id of the finance channel
+ * @param requestId the id if the reimbursement request
+ */
+export const sendReimbursementRequestCreatedNotification = async (requestId: string): Promise<void> => {
+  if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
+
+  const msg = `A reimbursement request has been created.`;
```

Comment:
we should change this so it uses the submitter name: 'Submitter-name created a reimbursement request :heavy_dollar_sign:`